openadapt/config.py

```diff
@@ -62,13 +62,13 @@
         # 'PHONE_NUMBER',
         # 'US_ITIN',
         # 'AU_ABN',
-        "DATE_TIME",
+        # "DATE_TIME",
```

Comment:
Uncomment DATE_TIME and URL entities as we no longer worry about the redaction of timestamps in visualization and also we want the most accurate scrubbing form presidio for testing.